database/migrations/2019_02_03_133007_change_id_type.php

```diff
@@ -30,31 +30,25 @@ public function up()
 
 	    // copy
 	    Photo::where('album_id_save',null)->update([
-			    "album_id_save" => DB::raw("`album_id`"),
+			    'album_id_save' => DB::raw('album_id'),
 	    ]);
 	    Album::where('parent_id_save',null)->update([
-		    "parent_id_save" => DB::raw("`parent_id`"),
+		    'parent_id_save' => DB::raw('parent_id'),
 	    ]);
 		DB::table('user_album')->where('album_id_save',null)->update([
-			"album_id_save" => DB::raw("`album_id`"),
+			'album_id_save' => DB::raw('album_id'),
 		]);
 
 	    Schema::table('user_album', function (Blueprint $table) {
-			if (DB::getDriverName() !== 'sqlite') {
-				$table->dropForeign(['album_id']);
-			}
+		    $table->dropForeign(['album_id']);
 		    $table->dropColumn('album_id');
 	    });
 	    Schema::table('photos', function (Blueprint $table) {
-			if (DB::getDriverName() !== 'sqlite') {
-				$table->dropForeign(['album_id']);
-			}
+		    $table->dropForeign(['album_id']);
```

Comment:
Can you please add back the check for `sqlite` here ? :)

Reply:
doesnt this become a duplicate entry then? for all 3 in this file

Reply:
Fixed.